### DISCUSSION TITLE
is there a way to use :has selector without arbitrary values, e.g. setting them up in config file

### DISCUSSION THREAD

**Discussion by bobbymannino** [tailwindlabs/tailwindcss#10307] [Help] [answered]
is there a way to use the has selector without using arbitrary values everywhere, is there a way to set it up in the config file something like 'has:hover:bg-red-500'?

**Comment by wongjn** [comment, marked as answer]
Yep! [Use a Tailwind plugin that adds `has` as a variant](https://tailwindcss.com/docs/plugins#adding-variants).
